test/mainnet-fork/4626Calls.t.sol

```diff
@@ -166,10 +171,14 @@ contract MainnetControllerWithdrawERC4626Tests is SUSDSTestBase {
         assertEq(susds.totalAssets(),                SUSDS_TOTAL_ASSETS + 1e18);
         assertEq(susds.balanceOf(address(almProxy)), SUSDS_CONVERTED_SHARES);
 
+        uint256 rateLimitBeforeWithdraw = rateLimits.getCurrentRateLimit(depositKey);
```

Comment:
Should add assertions for both withdraw and deposit rate limits, withdraw looks like its missing. Use real values for all no queries.